<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundForm</class>
 <widget class="QDialog" name="BackgroundForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="coloredRadioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="simpleTextRadioButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Simple Picture</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="tiledTextRadioButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Tiled Picture</string>
   </property>
  </widget>
  <widget class="QComboBox" name="spriteSimpleTextureComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>77</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="colorContainer" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>601</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>566</width>
      <height>140</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="TopLeftAngleLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Top Left</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLRSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLGSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLBSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLASpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Top Right</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLRSpinBox_4">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLGSpinBox_4">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLBSpinBox_4">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLASpinBox_4">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Bottom Right</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLRSpinBox_3">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLGSpinBox_3">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLBSpinBox_3">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLASpinBox_3">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Bottom Left</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLRSpinBox_2">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLGSpinBox_2">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLBSpinBox_2">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="TLASpinBox_2">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="spriteTiledTextureComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>77</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="simpleTextureAndTiledTextRadioButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Simple and Tiled Picture</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="colorAndTiledTextRadioButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Color and Tiled Picture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
